.pre-commit-config.yaml

```diff
@@ -31,4 +31,4 @@ repos:
       hooks:
       -   id: mypy
           args: [--config-file, pyproject.toml]
-          additional_dependencies: [numpy, pytest, zfpy, 'zarr==3.0.0b1']
+          additional_dependencies: [numpy, pytest, crc32c, zfpy, 'zarr==3.0.0b1']
```

Comment:
Would have thought adding `crc32c` here would be sufficient to fix the issue, but it keeps coming up

Am curious what we need to do to tell mypy it is ok to optionally use `crc32c`